js/ui/speed_dial_action/speed_dial_item.js

```diff
@@ -68,6 +82,22 @@ const SpeedDialItem = Overlay.inherit({
         }
     },
 
+    _getActionComponent() {
+        return this.option("actionComponent") || this.option("actions")[0];
+    },
+
+    _renderEvents() {
+        this._actionEvents.forEach((actionEvent) => {
+            this.on(actionEvent, () => {
+                this._getActionComponent()._createActionByOption("on" + actionEvent)(
```

Comment:
We pass 'oninitialized' here but check 'onInitialized' here https://github.com/DevExpress/DevExtreme/pull/8884/files#diff-645315b92c27d92aebe52d8df754b0e0R104 (lower case in the first case and the camel in the second). This is right?